cli/src/main/java/org/opentosca/toscana/cli/ApiController.java

```diff
@@ -49,7 +52,6 @@ public String uploadCsar(File file) {
         } catch (IOException e) {
             System.err.println(String.format(SOMETHING_WRONG + Constants.CSAR_UPLOAD_IO_ERROR
                 + Constants.ERROR_PLACEHOLDER, e.getMessage()));
-            e.printStackTrace();
```

Comment:
Removing stacktraces which provide important information is nooutpout reformating?

Reply:
I removed the Stacktraces because a shorter version of the error is shown right before the stacktrace and the stacktrace is only useful for someone who understands it. 
We can put it back in, or maybe make it a comment and someone who wants the additional information uncomments it.

Reply:
You could show stracktraces when the user calls the cli with the verbose-flag.